mllm/backends/qnn/CustomLayers.hpp

```
@@ -0,0 +1,72 @@
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "mllm/backends/base/PluginInterface.hpp"
+#include "mllm/core/DataTypes.hpp"
+#include "mllm/nn/Layer.hpp"
+
+// -------------------- Custom QNN Layers --------------------
+namespace mllm::nn::qnn {
+
+struct DequantizeAddOpOptions : public BaseOpOptions<DequantizeAddOpOptions> {
```

Clang (14.0.6)

[error] constructor does not initialize these fields: dtype, out_channels

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)